core-processor/src/executor.rs

```diff
@@ -259,7 +259,7 @@ pub fn execute_wasm<E>(
 where
     E: Environment,
     E::Ext: ProcessorExternalities + BackendExternalities + 'static,
-    <E::Ext as Externalities>::Error: BackendExternalitiesError,
+    <E::Ext as Externalities>::UnrecoverableError: BackendExternalitiesError,
```

Comment:
any reason for not defining same bound for `<E::Ext as Externalities>::FallibleError`?

Reply:
It seems these bounds are not actually required